### DISCUSSION TITLE
Standard torchvision models.

### DISCUSSION THREAD

**Discussion by NiklasGustafsson** [xamarin/TorchSharp#596] [Ideas]
If anyone is looking for ideas on what to help with, adding more of the standard models from 'torchvision.models' would be excellent.

I started on some of it already but porting the code from Python take a bit of time, so if anyone is interested in helping out, that would be excellent.

Niklas

**Comment by GeorgeS2019** [comment]
# PyTorch Image and Video => Torchvision

## [TORCHVISION.MODELS](https://pytorch.org/vision/0.11/models.html?msclkid=1175f46fce8311ecae6f8dd13689c62d#torchvision-models)
### [Classification](https://pytorch.org/vision/0.11/models.html?msclkid=1175f46fce8311ecae6f8dd13689c62d#classification)

- [X] [AlexNet](https://arxiv.org/abs/1404.5997)
- [X] [VGG](https://arxiv.org/abs/1409.1556)
- [X] [ResNet](https://arxiv.org/abs/1512.03385)
- [ ] [SqueezeNet](https://arxiv.org/abs/1602.07360)
- [ ] [DenseNet](https://arxiv.org/abs/1608.06993)
- [X] [Inception](https://arxiv.org/abs/1512.00567) v3
- [X] [GoogLeNet](https://arxiv.org/abs/1409.4842)
- [ ] [ShuffleNet](https://arxiv.org/abs/1807.11164) v2
- [ ] [MobileNetV2](https://arxiv.org/abs/1801.04381)
- [ ] [MobileNetV3](https://arxiv.org/abs/1905.02244)
- [ ] [ResNeXt](https://arxiv.org/abs/1611.05431)
- [ ] [Wide ResNet](https://pytorch.org/vision/0.11/models.html?msclkid=1175f46fce8311ecae6f8dd13689c62d#wide-resnet)

- [ ] [MNASNet](https://arxiv.org/abs/1807.11626)
- [ ] [EfficientNet](https://arxiv.org/abs/1905.11946)

- [ ] [RegNet](https://arxiv.org/abs/2003.13678)

### [Semantic Segmentation](https://pytorch.org/vision/0.11/models.html?msclkid=1175f46fce8311ecae6f8dd13689c62d#semantic-segmentation)
### [Object Detection, Instance Segmentation and Person Keypoint Detection](https://pytorch.org/vision/0.11/models.html?msclkid=1175f46fce8311ecae6f8dd13689c62d#object-detection-instance-segmentation-and-person-keypoint-detection)
### [Video classification](https://pytorch.org/vision/0.11/models.html?msclkid=1175f46fce8311ecae6f8dd13689c62d#video-classification)

**Comment by GeorgeS2019** [comment]
@NiklasGustafsson  What are Standard torchvision models?

**Reply by NiklasGustafsson** [reply]
https://pytorch.org/vision/stable/models.html

**Comment by GeorgeS2019** [comment]
@NiklasGustafsson  I think we are on the same page
